benchmark/stats/stats.go

```diff
@@ -85,7 +85,8 @@ func (stats *Stats) maybeUpdate() {
 	stats.histogram = NewHistogram(HistogramOptions{
 		NumBuckets: numBuckets,
 		// max-min(lower bound of last bucket) = (1 + growthFactor)^(numBuckets-2) * baseBucketSize.
-		GrowthFactor:   math.Pow(float64(stats.max-stats.min), 1/float64(numBuckets-2)) - 1,
+		// GrowthFactor can't be 0
+		GrowthFactor:   math.Pow(float64(stats.max-stats.min), 1/float64(numBuckets-2)) - 0.999,
```

Comment:
When (stat.max-stat.min = 1). There is a parameter oneOverLogOnePlusGrowthFactor: 1 / math.Log(1+opts.GrowthFactor), which mean GrowthFactor can't be 0. I can also add a 0.001 to oneOverLogOnePlusGrowthFactor but it is the same.
The reason it happens is because the latency is divided by a int64 time unit. For example,  If adding 40ms latency to the network, it may take 80.xxx ms for a round trip, so the final result only contains 80ms(stat.min) and 81ms(stat.max).